src/hotspot/share/logging/logConfiguration.cpp

```diff
@@ -559,6 +579,15 @@ void LogConfiguration::print_command_line_help(outputStream* out) {
 
   LogTagSet::describe_tagsets(out);
 
+  out->print_cr("\nAvailable log output options:");
```

Comment:
It doesn't seem right to me to have a section on output options first, but the way the rest of this section is structured makes it awkward to ordering things appropriately. Ideally, to me, we would have this order:
- Available log outputs
- Available log output options
- Additional file output options